test/test-creator-creator.js

```diff
@@ -18,79 +18,67 @@
 
 import * as common from './common';
 import * as testData from '../data/test-data.js';
-import chai from 'chai';
-import chaiAsPromised from 'chai-as-promised';
 import orm from './bookbrainz-data';
 import rewire from 'rewire';
 
 
-chai.use(chaiAsPromised);
-const {expect} = chai;
-
 const Achievement = rewire('../lib/server/helpers/achievement.js');
 
 const thresholdI = 1;
 const thresholdII = 10;
 const thresholdIII = 100;
 
+function rewireTypeCreation(threshold) {
+	return common.rewireAchievementTypeCreation(Achievement, 'creator', threshold);
+}
+
+function generate() {
+	return common.generate(Achievement, orm, true, 'creatorCreator');
+}
+
+function generateLabeled(rev) {
```

Comment:
This needs a clearer name and/or documentation. What is it generating and what is labelled?